ecloud/resource_instance.go

```diff
@@ -49,21 +50,23 @@ func resourceInstance() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
-			"volume_capacity": {
+			"os_volume_capacity": {
 				Type:     schema.TypeInt,
```

Comment:
Not sure if just leaving this as volume_capacity and inferring this as os volume would be better?

Reply:
I wanted to differentiate between the `volume_capacity` returned by the instances endpoint, as that value is dynamic based on the sum of all attached disks. The `os_volume_capacity` is there to make it clear that only os volume capacity is being updated. 
 What do you think?